ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/stun.j2

```diff
@@ -24,5 +27,16 @@ stun
 {%         if stun.server.local_interface is arista.avd.defined %}
       local-interface {{ stun.server.local_interface }}
 {%         endif %}
+{%         if stun.server.bindings_timeout is arista.avd.defined %}
+      binding timeout {{ stun.server.bindings_timeout }} seconds
+{%         endif %}
+{%         if stun.server.ssl_profile is arista.avd.defined %}
+      ssl profile {{ stun.server.ssl_profile }}
+{%         endif %}
```

Comment:
as mentioned on the molecule output - these two if may need to be inverted but can you check on your more recent EOS version to make sure?